cli/loadCompletion.sh

```
@@ -0,0 +1,43 @@
+    # End added
+  }
+  complete -F _opfab_completion opfab
+elif type compctl &>/dev/null; then
+  _opfab_completion () {
+    local cword line point si
+    read -Ac words
+    read -cn cword
+    read -l line
+    si="$IFS"
+    if ! IFS=$'
+' reply=($(opfab --compzsh --compgen "${cword}" "${words[cword-1]}" "${line}")); then
```

Shellcheck

[warning] reply appears unused. Verify use (or export if used externally).

(SC2034)